fdbrpc/tests/AuthzTlsTest.actor.cpp

```diff
@@ -36,46 +41,64 @@
 
 std::FILE* outp = stdout;
 
+enum ChainLength : int {
```

Comment:
Can we please use `enum class` instead of `enum`?

Reply:
Normally I would strongly agree with you. However, this is not a usual `strictly choose among n` enum use-case. The values of the `enum ChainLength` could be anywhere as long as it's a reasonably small positive/negative integer plus the `NO_TLS` value. The convertibility from `enum` to `int`, which normally considered bad, is actually desired in this case because the integral value actually means something. Also I wanted `int` with a special formatting output beyond its numeric value (for the `NO_TLS` case). I'll address accordingly when my argument no longer holds.

Reply:
Now that you mention it, a `struct` with a `operator int()` could have been a nicer way to go. I'll address it in any future modification.